sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/ApproximateUnique.java

```diff
@@ -42,6 +42,9 @@
 /**
  * {@code PTransform}s for estimating the number of distinct elements in a {@code PCollection}, or
  * the number of distinct values associated with each key in a {@code PCollection} of {@code KV}s.
+ *
+ * <p>Consider using {@code HllCount} in the {@code zetasketch} extension module if you need better
+ * performance or need to save intermediate aggregation result into a sketch for later processing.
```

Comment:
Could you please a code example here about how to use your transform instead of `ApprocimateUnique` and attach your design doc, especially `Comparison with existing Beam transforms: ApproximateUnique & ApproximateDistinct` section.

Reply:
Done.